From 1c6e0962dc1294b4b28513449aa530f14169fa09 Mon Sep 17 00:00:00 2001
From: Han Xu <han.xu@nxp.com>
Date: Mon, 27 Nov 2017 15:17:41 -0600
Subject: [PATCH 2905/5242] MLK-17006: arm64: dts: fix the imx8qxp nand pin
 conflict with usdhc1

commit  2b8e546b7330e3c1f6dfb75616627b9d344fbd94 from
https://source.codeaurora.org/external/imx/linux-imx.git

fix the pin conflict between nand and usdhc1 on imx8qxp validation
board.

BuildInfo:
 - SCFW daea284c, IMX-MKIMAGE 90fbac1a, ATF
 - U-Boot 2017.03-00713-g345bcc2

Signed-off-by: Han Xu <han.xu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../fsl-imx8qxp-lpddr4-arm2-gpmi-nand.dts          |    6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-gpmi-nand.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-gpmi-nand.dts
index b85c69d..3169b61 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-gpmi-nand.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-gpmi-nand.dts
@@ -56,7 +56,11 @@
 	nand-on-flash-bbt;
 };
 
-/* Disabled the usdhc2 since pin conflict */
+/* Disabled the usdhc1/usdhc2 since pin conflict */
+&usdhc1 {
+	status = "disabled";
+};
+
 &usdhc2 {
 	status = "disabled";
 };
-- 
1.7.9.5

